packages/components/src/inputs/src/InputGroup.module.css

```diff
@@ -0,0 +1,92 @@
+.hop-InputGroup  {
+    --input-group-background: var(--hop-neutral-surface);
+    --input-group-border: var(--hop-neutral-border);
+
+    position: relative;
+
+    display: inline-flex;
+    gap: var(--hop-space-inline-sm);
+    align-items: center;
+
+    box-sizing: border-box;
+    inline-size: fit-content;
+    block-size: 2.5rem;
+    padding-inline: var(--hop-space-inset-md) var(--hop-space-inset-md);
+
+    background: var(--input-group-background);
+    border: 0.0625rem solid var(--input-group-border);
```

Comment:
could have a border size variable.
also, the border variable should specify color.
so it would probably be --hop-InputGroup-border-color and the internal one would be --border-color